.eslintrc

```diff
@@ -11,9 +11,7 @@
   "ecmaFeatures": {
     "modules": false
   },
-  "extends": [
-    "airbnb-base"
-  ],
+  "extends": ["airbnb-base"],
```

Comment:
Can you axe this? Just merged @asudoh's PR that brings in `carbon-base` (which consumes airbnb-base + prettier)